client/src/components/Shared/Form/index.js

```diff
@@ -33,58 +115,141 @@ class Form extends Component {
         <StyledInformation>PIECE INFORMATION</StyledInformation>
         <StyledNotic>Please verify the items marked* </StyledNotic>
 
-
         <StyledDiv>
-
           <StyledLabels>
-
-            <StyledLabel> Item Type* </StyledLabel>
-            <StyledLabel> Color* </StyledLabel>
-            <StyledLabel> Brand </StyledLabel>
-            <StyledLabel> Other </StyledLabel>
-            <StyledLabel> Condition </StyledLabel>
-            <StyledLabel> Label size </StyledLabel>
-            <StyledLabel> Age </StyledLabel>
-            <StyledLabel> purchase price </StyledLabel>
-
+            {this.state.categories.map(category => {
+              return <StyledLabel key={category}>{category}</StyledLabel>;
+            })}
           </StyledLabels>
 
           <StyledItem>
-
-            <StyledSelect>
-              <StyledOption value="T-shirt">T-shirt</StyledOption>
+            <StyledSelect
```

Comment:
As this code is repeated quite a few times, I'd try to abstract it out into a component. This will also make the file easier to read/review.